### PR TITLE
quick fix of documentation mathematics

### PR DESCRIPTION
**Pull request by condnsdmatters**
Boring but important doc error

### CODE REVIEW

File: gpflow/conditionals/util.py [hunk 1]
```diff
@@ -15,8 +15,10 @@ def base_conditional(Kmn: tf.Tensor,
     """
     Given a g1 and g2, and distribution p and q such that
       p(g2) = N(g2; 0, Kmm)
+
       p(g1) = N(g1; 0, Knn)
-      p(g1 | g2) = N(g1;0,Knm)
+
+      p(g1 | g2) = N(g1; Knm(Kmm^-1)g2, Knn - Knm(Kmm^-1)Kmn)
```
Comment (st--) [inline gpflow/conditionals/util.py]:
```suggestion
      p(g1 | g2) = N(g1; Kₙₘ Kₘₘ⁻¹ g2, Kₙₙ - Kₙₘ Kₘₘ⁻¹ Kₘₙ)
```
(would be nice updating the other maths with unicode as well but not necessary for this PR!)

File: gpflow/conditionals/util.py [hunk 2]
```diff
@@ -15,8 +15,10 @@ def base_conditional(Kmn: tf.Tensor,
     """
     Given a g1 and g2, and distribution p and q such that
       p(g2) = N(g2; 0, Kmm)
+
       p(g1) = N(g1; 0, Knn)
```
Comment (st--) [inline gpflow/conditionals/util.py]:
```suggestion
      p(g1) = N(g1; 0, Kₙₙ)
```

File: gpflow/conditionals/util.py [hunk 3]
```diff
@@ -15,8 +15,10 @@ def base_conditional(Kmn: tf.Tensor,
     """
     Given a g1 and g2, and distribution p and q such that
       p(g2) = N(g2; 0, Kmm)
```
Comment (st--) [inline gpflow/conditionals/util.py]:
```suggestion
      p(g2) = N(g2; 0, Kₘₘ)
```

File: gpflow/conditionals/util.py (hunk 1 above)
Comment (st--) [reply gpflow/conditionals/util.py]: Or just changing the inverse:
```suggestion
      p(g1 | g2) = N(g1; Knm (Kmm⁻¹) g2, Knn - Knm (Kmm⁻¹) Kmn)
```
given that `Kmm` is actually the variable name in the code...

File: gpflow/conditionals/util.py (hunk 3 above)
Comment (condnsdmatters) [reply gpflow/conditionals/util.py]: discussed and ignoring

File: gpflow/conditionals/util.py (hunk 2 above)
Comment (condnsdmatters) [reply gpflow/conditionals/util.py]: discussed and ignoring

**Pull request merged**